### PR TITLE
Add CI/CD to the repo

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
### Overview

In this PR I add the configuration for running the unit tests of the code in Linux with the Github actions. I also set up the labeler and the dependabot. These configuration files are modified versions of the ones in `PyVista` [repository](https://github.com/pyvista/pyvista/tree/main/.github).

### CODE REVIEW

**Comment by AlejandroFernandezLuces** [comment]
@MaxJPRey could you take a look if you are available?

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+maintenance:
+- .github/**/*
+- codecov.yml
+- .flake8
+- .coveragerc
+- .codecimate.yml
+- Makefile
+- ignore_words.txt
+- mypy.ini
+- pyproject.toml
```
Comment (MaxJPRey) [inline .github/labeler.yml:16]:
```suggestion
- pyproject.toml
- setup.py
```

Comment (MaxJPRey) [reply .github/labeler.yml:16]: We will remove `setup.py` as soon as we transition toward `pyproject.toml`.

File: .github/labeler.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- examples_flask/**/*
+- ./*.md
+- ./*.rst
+maintenance:
+- .github/**/*
+- codecov.yml
+- .flake8
+- .coveragerc
+- .codecimate.yml
+- Makefile
```
Comment (MaxJPRey) [inline .github/labeler.yml:13]:
```suggestion
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

File: setup.py [hunk 3]
```diff
@@ -41,7 +41,7 @@ def read(fname):
     ],
     entry_points={
         "pytest11": [
-            "pyvista = pytest_pyvista",
+            "pyvista = pytest_pyvista.pytest_pyvista",
```
Comment (MaxJPRey) [inline setup.py:44]: As discussed, please add some explanations.

**Pull request merged**